### ISSUE TITLE
Wrong Content-Disposition on attachments (size=0)

### ISSUE THREAD

**Issue by fauvetg** [bbottema/simple-java-mail#103]
Hello,

When building a `MimeMessage` from an `Email` with the class `MimeMessageHelper`, the resulting `MimeMessage` has a wrong Content-Disposition header for attachments. The size is attribute is not correct : 

```
------=_Part_32_1708476334.1507045824508
Content-Type: application/pdf; filename=XXX.pdf; name=XXX
Content-Transfer-Encoding: base64
Content-Disposition: attachment; size=0; filename=XXX.pdf
Content-ID: <ATT_TE91266914>

...
```

Some mailers (reproduced on and Apple mailer and Horde) cannot open those attachments.

I don't have a fix to provide, but I think the bug is in method `MimeMessageHelper#getBodyPartFromDatasource`. 

NB : we use version 4.2.1

**Comment by bbottema** [comment]
I'll look into it. Thanks for the report!

**Comment by bbottema** [comment]
Released as 5.0.0.rc1-SNAPSHOT. Add [OSS' snapshots repo](https://stackoverflow.com/questions/7715321/how-to-download-snapshot-version-from-maven-snapshot-repository/7717234#7717234) to find it in Maven.
